packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue

```diff
@@ -11,8 +11,8 @@
 		</div>
 		<p>
 			Set the {{ dynamicInterventions[0].type }}&nbsp; <strong>{{ dynamicInterventions[0].appliedTo }}</strong> to
-			<strong>{{ dynamicInterventions[0].threshold }}</strong> days when it crosses the threshold value
-			<strong>{{ dynamicInterventions[0].value }}</strong> person.
+			<strong>{{ dynamicInterventions[0].value }}</strong> when it crosses the threshold value
+			<strong>{{ dynamicInterventions[0].threshold }}</strong>
```

Comment:
So these are the wording changes, where are the unit changes you mentioned in the description, are they already merged via another PR?

Reply:
never mind, realized you meant the hardwired values

Reply:
`days` and `person` are the units I am referring to in the PR description 
These are assumed but not necessarily accurately